Cargo.toml

```diff
@@ -13,6 +13,7 @@ fraction = "0.13.1"
 libc = "0.2"
 paste = "1.0"
 rand = "0.8.5"
+rand_distr = "0.4.3"
```

Comment:
could using this crate also be useful for our uniform implementation, I have seen, that this also offers uniform sampling.
Have you looked into it, and whether this could improve our implementation in any way shape or form?

Reply:
The uniform distribution is also available in the rand-crate, but it does not allow to sample values larger than u64. For binomial sampling, this is currently also the case, but I plan on expanding the implementation later on to be more versatile. For now, I believe that sampling a maximum of u64 number of tries is sufficient.